### ISSUE TITLE
When I run astro airflow deploy <name> it fails.

### ISSUE THREAD

**Issue by andriisoldatenko** [astronomer/astro-cli#202]
but when I run `astro airflow deploy` and then choose from the list

**Comment by andriisoldatenko** [comment]
can't reproduce.
cc @schnie 
```
./astro-v082 airflow deploy ultraviolet-spectroscope-8451
Deploying: ultraviolet-spectroscope-8451
ultraviolet-spectroscope-8451/airflow
Building image...
Sending build context to Docker daemon  36.96MB
Step 1/1 : FROM astronomerinc/ap-airflow:latest
 ---> 5719f204898d
Successfully built 5719f204898d
Successfully tagged ultraviolet-spectroscope-8451/airflow:latest
Pushing image to Astronomer registry
The push refers to repository [registry.alpha.datarouter.ai/ultraviolet-spectroscope-8451/airflow]
56a36fbf02b3: Pushed
28c5cb28b032: Pushed
fb395556d9a8: Pushed
2657f4bf0931: Pushed
740c88129889: Pushed
2a6902fcd063: Pushed
df64d3292fd6: Pushed
```

I think i can add more examples ?

**Comment by schnie** [comment]
Just caught up on this with @ryw. I think its working as intended. Do we need to update docs or anything to avoid confusion?

**Comment by andriisoldatenko** [comment]
yep I think add example + replace DEPLOYMENT -> DEPLOYMENT NAME?

**Comment by andriisoldatenko** [comment]
@ryw 
>Yeah but the response could say “are you sure you’re using the Deployment Name?”

**Comment by schnie** [comment]
Yea, that's probably a good idea - a friendly, suggestive error message. We call it "release name" (the helm terminology) in the code and in Orbit, so maybe use that? 

Orbit has a "Name" field for the deployment, but it refers to the friendly, user-changable name, which could be interpreted as the "Deployment name", which is known as "label" in the code I believe.

I guess we kind of have some terminology we need to clarify on our end. @ryw @benjamingregory @vparekh94 @ashb @andriisoldatenko you guys have any thoughts about clarifying and standardizing our terminology? I think we just have to clarify what we call the friendly, user-changeable field, and what we call the non-changeable, space-words, helm field.

"Deployment name" or "Deployment label" or ...
and
"Relase name" or "Release" or "Deployment release" or "Deployment release name" or ...
